src/simpleImgProvider.js

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import type { Context } from './simpleImg';
-import filterImgSrc from './logic/filterSrcset';
+import init from './init';
 
 export const APPEND_IMAGE_REF = '__ProgresssiveImagesAppendImageRef__';
```

Comment:
Is it really neccessary to use dangling underscores to name these special context methods? It makes it really hard to read, both here and in SimpleImg component.

Reply:
i followed egghead.io's tutorial from the paypal developer, maybe it's an over kill.